exporters/zipkin/CMakeLists.txt

```diff
@@ -14,9 +14,12 @@
 
 include_directories(include)
 find_package(CURL)
-if(CURL_FOUND)
-  add_definitions(-DWITH_CURL)
+if(NOT CURL_FOUND)
```

Comment:
how about using `find_package(CURL REQUIRED)`?

Reply:
Yes, this looks better.